src/test/java/games/strategy/ui/SwingWorkerCompletionWaiterTest.java

```diff
@@ -1,24 +1,20 @@
 package games.strategy.ui;
 
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 
 import java.beans.PropertyChangeEvent;
 
 import javax.swing.SwingWorker;
 
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(MockitoJUnitRunner.class)
 public final class SwingWorkerCompletionWaiterTest {
```

Comment:
#2077 has been merged.  It will probably be easiest to just revert all changes to this file, and then merge `master` into this branch.